setup.py

```diff
@@ -62,7 +62,7 @@ def read_content(filepath):
 
 setup(
     name="iiblib",
-    version="1.0.0",
+    version="2.0.0",
```

Comment:
This shouldn't be changed yet.

Reply:
This has to be changed here since we have pidiff in test. Otherwise CI will fail.